data/language/en-GB.txt

```diff
@@ -3766,6 +3766,9 @@ STR_6315    :{WINDOW_COLOUR_2}Pathfind Goal: {BLACK}{INT32}, {INT32}, {INT32} di
 STR_6316    :{WINDOW_COLOUR_2}Pathfind history:
 STR_6317    :{BLACK}{INT32}, {INT32}, {INT32} dir {INT32}
 STR_6318    :Network desync detected.{NEWLINE}Log file: {STRING}
+STR_6319    :{WINDOW_COLOUR_2}Block Brake Closed
+STR_6320    :{WINDOW_COLOUR_2}Indestructible
+STR_6321    :Broken
```

Comment:
The path is not broken, the path addition is.

Also, this lacks the `{WINDOW_COLOUR_2}` formatting code.

```suggestion
STR_6321    :{WINDOW_COLOUR_2}Addition is broken
```